hack/deploy.sh

```diff
@@ -30,13 +30,13 @@ fi
 kubectl apply -f ./ServiceAccount.yaml
 kubectl apply -f ./Identity.yaml
 kubectl apply -f ./Service.yaml
-kubectl apply -f ./Deployment.yaml
+kubectl apply -k ./kustomizations/remote/istanbul
```

Comment:
“Istanbul” is the codename for distroless Aegis images.

“Photon” is the codename for VMware Photon OS 5.0 Aegis images.